gpii/configs/gpii.config.cloudBased.development.all.local.txt

```diff
@@ -11,7 +11,7 @@ This mean that:
 ** OntologyHandler
 ** rawPreferencesServer (reading preferences from file)
 ** solutionsRegistry (reading registry from file)
-** flatMatchMaker
+** defaultMatchMaker
```

Comment:
Odd that this reads "defaultMatchMaker" whereas the all.development reads "canopyMatchMaker"